flink-kubernetes-mock-shaded/pom.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I completely removed the whole mock shaded module (https://github.com/gyfora/flink-kubernetes-operator/commit/db0fbcc3e97ce6c6c7eb9a4575ab1305fd26c34c) and everything seem to still work. All tests still pass.

Do we still need this? What am I missing?

Reply:
And it actually fixed some of my other intellij test issues